### ISSUE TITLE
disable setCurrent when calling an action

### ISSUE THREAD

**Issue by mrfrase3** [feathers-plus/feathers-vuex#186]
Sometimes I want to call a get/create action to the server without setting the result to the current context.

I suggest either adding an option to disable these calls, or remove them completely and let the developer have control over setting the current.

If you want me to implement it, I can add it to my current PR.

**Comment by marshallswain** [comment]
@mrfrase3 I agree that they can get in the way.  I'm planning on removing them from the actions.  Today I made a release that adds some more methods to each service's FeathersVuexModel class.  See the docs, here:  https://feathers-plus.github.io/v1/feathers-vuex/model-classes.html#Model-Methods

I feel like adding another method to the Model class, so you could call `Todo.setCurrent(idOrInstance)` 
 (which would call the mutation)  or you can always use the mutation, directly.

Removing the calls to setCurrent from the actions will be a breaking change, but my vote goes towards making it happen.

**Comment by marshallswain** [comment]
I'm just about ready to publish the pre-release to master.  I'm just going to do some production testing of these new features: https://feathers-plus.github.io/v1/feathers-vuex/model-classes.html#instanceDefaults-Function-pre-release.

**Comment by marshallswain** [comment]
This is implemented as a non-breaking option, now.  You can use `setCurrentOnGet` and `setCurrentOnCreate` booleans in the service state to control this functionality.  Docs have been updated:  https://feathers-plus.github.io/v1/feathers-vuex/service-module.html

`npm i feathers-vuex@1.7.0-pre.45`
